airflow/models/abstractoperator.py

```diff
@@ -395,6 +395,19 @@ def get_closest_mapped_task_group(self) -> MappedTaskGroup | None:
         """
         return next(self.iter_mapped_task_groups(), None)
 
+    def get_needs_expansion(self) -> bool:
+        """
+        Returns true of the task is MappedOperator or is in a mapped task group.
+
+        :meta private:
+        """
+        if self._needs_expansion is None:
+            if self.get_closest_mapped_task_group() is not None:
+                self._needs_expansion = True
+            else:
+                self._needs_expansion = False
+        return self._needs_expansion
```

Comment:
```suggestion
        if self._needs_expansion is not None:
            return self._needs_expansion
        if self.get_closest_mapped_task_group() is not None:
            needs_expansion = True
        else:
            needs_expansion = False
        self._needs_expansion = needs_expansion
        return needs_expansion
```

I think this is needed to get past Mypy…

Reply:
i think that there was actually no mypy complaint, just a ruff complaint about the docstring "mood"